jujugui/static/gui/src/app/components/inspector-header/test-inspector-header.js

```diff
@@ -0,0 +1,94 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+var juju = {components: {}};
+var testUtils = React.addons.TestUtils;
+var renderIntoDocument = testUtils.renderIntoDocument;
+
+chai.config.includeStack = true;
+chai.config.truncateThreshold = 0;
+
+function queryComponentSelector(component, selector, all) {
+  var queryFn = (all) ? 'querySelectorAll' : 'querySelector';
+  return component.getDOMNode()[queryFn](selector);
+}
+
+describe('InspectorHeader', function() {
+
+  beforeAll(function(done) {
+    // By loading this file it adds the component to the juju components.
+    YUI().use('inspector-header', function() { done(); });
+  });
+
+  it('displays the provided title', function() {
+    var component = renderIntoDocument(
+        <juju.components.InspectorHeader
+          title="Juju GUI" />);
+    assert.equal(
+        queryComponentSelector(
```

Comment:
I think this line break is at the wrong spot, should be after `innerText` if possible